### DISCUSSION TITLE
What is the most optimise way of integrating Shad/cn-ui as workspace.

### DISCUSSION THREAD

**Discussion by shagunsharma11** [vercel/turborepo#7693] [Help] [answered]
### Summary

There are very few resources available to explain the integration of Shadcn into the workspace. 

It would be better if provided in the example folder of this repository.

**Comment by anthonyshew** [comment]
I am currently working on a refresh of our documentation that will include shadcn/ui documentation. Sorry for not having this today - but stay tuned!

**Comment by nicck1** [comment]
Great to hear that you are already addressing this gap. In the meantime, I have created this template for public use. Can I raise a PR to add this to the examples folder? @anthonyshew 

Github - https://github.com/shagunsharma6677/turborepo-shadcn-ui-template

**Comment by sheik-mostafizur** [comment]
Can anyone help me integrate shadcn-ui? I can not get proper solutions.

Issue - https://github.com/sheik-mostafizur/turbo-frontend/issues/1

**Comment by anthonyshew** [comment, marked as answer]
shadcn/ui has added monorepo support! https://turbo.build/repo/docs/guides/tools/shadcn-ui

We also recommend reading their docs.
